### DISCUSSION TITLE
Any reason public template configured as desktop-first?

### DISCUSSION THREAD

**Discussion by ayhanap** [maizzle/framework#1022] [Ideas]
I found out that public-template has the following in `tailind.config.js`
```
 screens: {
    xs: {max: '425px'},
    sm: {max: '600px'},
  },
```

Which is the opposite of Tailwind CSS. I found this to be counter-intuitive from a framework utilizing Tailwind. I just removed those and altered the mail templates, but it is initially confusing.

Is there any technical reason for this that we need to be aware of?

Also, the [https://maizzle.com/docs/introduction#responsive](documentation) mentions it wrong by saying Tailwind is desktop-first. 

> Tailwind CSS screens are configured for a **desktop-first** responsive approach, which is the opposite of what you might be used to in web development.

**Comment by cossssmin** [comment]
Mainly because of Outlook on Windows and other email clients that do not support media queries.

> Also, the [...] mentions it wrong by saying Tailwind is desktop-first.

I agree it could be more clear by saying that _in Maizzle, Tailwind CSS screens..._.

**Reply by cossssmin** [reply]
Updated docs: 

https://maizzle.com/docs/introduction#responsive
